docs/ANTIGRAVITY_DOM_SELECTORS.md

```
@@ -8,13 +8,17 @@ Central reference for all CSS selectors and DOM structures used to interact with
 
 ## Root Scope
 
-All scripts scope queries to the side panel first, falling back to `document`.
+Most scripts scope queries to a specific container first.
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)